app/localize.js

```diff
@@ -0,0 +1,27 @@
+var languages = {
```

Comment:
I think translations should be stored in the config directory (same than dictionaries, actually) and loaded dynamically. It makes it easier for user to add custom languages.